### PR TITLE
Move hardware blogs to docs

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by Hasmin-AC — APPROVED** [review]

File: src/node-red/hardwares/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 eleventyNavigation:
-  key: Hardware
+  key: Hardwares
```
Comment (ZJvandeWeg) [inline src/node-red/hardwares/index.md:3]: Nope. This one isn't plural. 😅

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/node-red/hardwares/raspberry-pi-5.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    ```sh
+    bash <(curl -sL https://raw.githubusercontent.com/FlowFuse/device-agent/main/service/raspbian-install-device-agent.sh)
+    ```
+
+   This script installs a Node.js runtime, the agent, and configures the Raspberry Pi to run the FlowFuse agent on boot and restart it if it crashes.
+
+### Configuring the FlowFuse Agent
+
+To configure the FlowFuse Cloud with your devices, you need to set up a token for identification. Follow the instructions in the [FlowFuse documentation](/docs/user/introduction/#working-with-devices) to obtain and configure the device YAML file.
```
Comment (ZJvandeWeg) [inline src/node-red/hardwares/raspberry-pi-5.md:59]: Do we want to update the instructions to include the quick onboard command?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/node-red/hardwares/index.md (hunk 1 above)
Comment (sumitshinde-84) [reply src/node-red/hardwares/index.md:3]: I will update it.

File: src/node-red/hardwares/raspberry-pi-5.md (hunk 2 above)
Comment (sumitshinde-84) [reply src/node-red/hardwares/raspberry-pi-5.md:59]: You are referring to command provided in the configuration dialog after creating device, right?

Comment (ZJvandeWeg) [reply src/node-red/hardwares/raspberry-pi-5.md:59]: Indeed!

Comment (sumitshinde-84) [reply src/node-red/hardwares/raspberry-pi-5.md:59]: okay, I will make update.

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**